.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,83 @@
+name: allcontributors-auto-detect
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I wondered -- shouldn't it also be triggered by pull requests which touch the files relevant to `con/tributors`?  but then it would only be usable for PRs submitted from branches within this repo (just in case -- invited you @vsoch with Write permissions to this repo)

Reply:
I think we'd only want the PR opened for updates after an official merge. I had pull request originally for testing purposes only. I will add it back to see how / if it works now that I have write!